doc/_static/switcher.json

```diff
@@ -12,8 +12,8 @@
     },
     {
         "name": "3.9",
-        "version": "3.9.3",
-        "url": "https://matplotlib.org/3.9.3/"
+        "version": "3.9.4",
+        "url": "https://matplotlib.org/3.9.4/"
```

Comment:
This URL didn't exist, not sure why it got bumped on `v3.10.x`.

Reply:
Since the link is broken, is it worth revertig to the 3.9.3 link in this PR and opening a fresh issue about building and publishing the 3.9.4 docs?